src/declarations/stencil-public-compiler.ts

```diff
@@ -311,17 +311,6 @@ export interface ConfigExtras {
    */
   lifecycleDOMEvents?: boolean;
 
-  // TODO(STENCIL-663): Remove code related to deprecated `safari10` field.
-  /**
-   * Safari 10 supports ES modules with `<script type="module">`, however, it did not implement
-   * `<script nomodule>`. When set to `true`, the runtime will patch support for Safari 10
-   * due to its lack of `nomodule` support.
-   * Defaults to `false`.
-   *
-   * @deprecated Since Stencil v3.0.0, Safari 10 is no longer supported.
-   */
-  __deprecated__safari10?: boolean;
```

Comment:
There's a few references to the old property name I found in some of our tests. Can we remove these while we're in here?

I found a few references to the _old_ name (`cssVarsShim`, prior to the rename) in:

- https://github.com/ionic-team/stencil/blob/42314dc202e9832086207759565c6660d3746eff/test/jest-spec-runner/stencil.config.ts#L14
- https://github.com/ionic-team/stencil/blob/42314dc202e9832086207759565c6660d3746eff/test/todo-app/stencil.config.ts#L16
- https://github.com/ionic-team/stencil/blob/42314dc202e9832086207759565c6660d3746eff/test/karma/stencil.config.ts#L37